java/src/jmri/jmrit/beantable/signalmast/AddSignalMastPanel.java

```diff
@@ -73,21 +72,23 @@ public AddSignalMastPanel() {
             }
         );
         
-        { // scoping for temporary variables
-
-            String[] tempMastNamesArray = new String[panes.size()];
-            int i = 0;
-            for (SignalMastAddPane pane : panes) {
-                tempMastNamesArray[i++] = pane.getPaneName();
-            }
-            signalMastDriver = new JComboBox<>(tempMastNamesArray);
+        // scoping for temporary variables
+        String[] tempMastNamesArray = new String[panes.size()];
+        int i = 0;
+        for (SignalMastAddPane pane : panes) {
+            tempMastNamesArray[i++] = pane.getPaneName();
         }
+        signalMastDriver = new JComboBox<>(tempMastNamesArray);
+        init();
+    }
+    
+    final void init() {
 
         setLayout(new BoxLayout(this, BoxLayout.Y_AXIS));
 
         JPanel p;
         p = new JPanel();
-        p.setLayout(new jmri.util.javaworld.GridLayout2(5, 2));
+        p.setLayout(new GridLayout2(5, 2));
```

Comment:
I usually thinks it's great to import packages and only refer to the class when creating an instance. But in this particular case, I think it would be good to keep `jmri.util.javaworld.GridLayout2`, unless something warns about it. The reason is that `GridLayout` is a very well known class and it's so easy to miss that this is not an instance of GridLayout, but a different class. I wasn't even aware of this class.

Reply:
Agreed, this could easily be overlooked.  I wasn't aware of it either!